.github/workflows/ci.yml

```diff
@@ -44,15 +36,10 @@ jobs:
       run: |
         python -m pip install -r test_requirements.txt
         python -m pip install -e .
-        if [ -n "$PIP_EXTRA_INSTALLATION" ]; then
-          # deliberately not quoted
-          python -m pip install $PIP_EXTRA_INSTALLATION
-        fi
         python -m pip check
     - name: Lint
       run: |
-        python -m pip install flake8 flake8_formatter_junit_xml
-        mkdir -p test-reports/flake8
```

Comment:
Did these not work?

Reply:
Ah nevermind, the point was that CircleCI did some nice rendering when it found these junit files. Unfortunately github actions doesn't.